Assets/FallingPlatform.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FallingPlatform : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+    public GameObject spriteObject;
+    public float elapsed = 0.0f;
```

Comment:
Nit: Does this need to be serialized or accessed by an external script?

Reply:
Good catch , will switch all public variables to serialized .